### PR TITLE
Import repo logging

### PR DESCRIPTION
**Pull request by adamretter**
1. Log a warning if autodeploy dir is not found
2. Show Repo logs on the Docker stdout console

### CODE REVIEW

**Review by duncdrum — APPROVED** [review]

**Review by duncdrum — COMMENTED** [review]
if there is new info in the logs by default, could you add a test to make sure that it stays there going forward

**Comment by adamretter** [comment]
@duncdrum testing for log output is not something we have ever done and would be very hard to achieve. I think that is likely a step to far...

**Comment by duncdrum** [comment]
while we're not testing logging as such, we are and have bee reading the logs of the container, to ensure our modifications to the logging config are working as intended, if we add more modifications, we should make sure that the new messages are visible to the client.

**Comment by adamretter** [comment]
@duncdrum I won't be adding a test for this because I don't believe the time/effort trade-off is worth the value of the test.

Do you want me to close this PR, or are you happy to proceed without the test?

File: exist-core/src/main/java/org/exist/repo/AutoDeploymentTrigger.java [hunk 1]
```diff
@@ -77,6 +77,7 @@ public void execute(final DBBroker sysBroker, final Txn transaction, final Map<S
         }
 
         if (!Files.isReadable(autodeployDir) && Files.isDirectory(autodeployDir)) {
+            LOG.warn("Unable to read autodeploy directory: " + autodeployDir);
```
Comment (line-o) [inline exist-core/src/main/java/org/exist/repo/AutoDeploymentTrigger.java:80]: The proposed fix for the code smell reported by SonarCloud
`LOG.warn("Unable to read autodeploy directory: {}", autodeployDir)`

**Review by dizzzz — CHANGES_REQUESTED** [review]
LGTM, small logging item as requested

File: exist-core/src/main/java/org/exist/repo/AutoDeploymentTrigger.java (hunk 1 above)
Comment (dizzzz) [reply exist-core/src/main/java/org/exist/repo/AutoDeploymentTrigger.java:80]: It is consistent with style of rest of code.

**Comment by adamretter** [comment]
@line-o @dizzzz done.

**Review by dizzzz — APPROVED** [review]

**Pull request merged**